examples/pennylane_run_quantum_natural_gradient.py

```diff
@@ -0,0 +1,401 @@
+r"""
+
+.. _quantum_natural_gradient:
+
+Quantum natural gradient
+========================
+
+This example demonstrates the quantum natural gradient optimization technique
+for variational quantum circuits, originally proposed in
+`Stokes et al. (2019) <https://arxiv.org/abs/1909.02108>`__.
+
+Background
+----------
+
+The most successful class of quantum algorithms for use on near-term noisy quantum hardware
+is the so-called variational quantum algorithm. As laid out in the :ref:`Concepts <varcirc>`, in variational quantum algorithms
+a low-depth parametrized quantum circuit ansatz is chosen, and a problem-specific
+Hermitian observable measured. A classical optimization loop is then used to find
+the set of quantum parameters that *minimize* a particular measurement statistic
+of the quantum device. Examples of such algorithms include the :ref:`variational quantum
+eigensolver (VQE) <vqe>`, the `quantum approximate optimization algorithm (QAOA) <https://arxiv.org/abs/1411.4028>`__,
+and :ref:`quantum neural networks (QNN) <quantum_neural_net>`.
+
+Most recent implementations
+of variational quantum algorithms have used gradient-free classical optimization
+methods, such as Nelder-Mead. However, the parameter-shift rule for
+analytic quantum gradients (as implemented in PennyLane) has allowed for
+stochastic gradient descent of variational quantum algorithms on quantum
+hardware. One caveat though that has surfaced with gradient descent
+is the issue of learning rate or step-size --- how do we choose the optimal
+step size for our variational quantum algorithms, to ensure successful and
+efficient optimization?
+
+The natural gradient
+^^^^^^^^^^^^^^^^^^^^
+
+In standard gradient descent, each optimization step is given by
+
+.. math:: \theta_{t+1} = \theta_t -\eta \nabla \mathcal{L}(\theta),
+
+where :math:`\mathcal{L}(\theta)` is the loss as a function of
+the parameters :math:`\theta`, and :math:`\eta` is the learning rate
+or step size. In essence, each optimization step calculates a vector of
+steepest descent direction around the local value of :math:`\theta_t`
+in the parameter space, and updates :math:`\theta_t\rightarrow \theta_{t+1}`
+by this vector.
+
+The problem with the above approach is that each optimization step
+is strongly connected to a *Euclidean geometry* on the parameter space.
+The parametrization is not unique, and different parametrizations can distort
+distances within the optimization landscape.
+
+For example, consider the following loss function:
+
+.. math:: \mathcal{L}(x, y) = 5(1- e^{-(x^2 + (y/10)^2)})
+
+We wish to find the values of parameters :math:`x` and :math:`y` that minimize
+the loss. Here, movement in the :math:`x` direction results in significantly larger changes

```

Comment:
This is not clear to me.  as the change in x direction is larger, the value of x changes more than that of y in each optimization step. so you WANT (not expect) that the increment in x is smaller in order for the whole function to converge nicely?

Reply:
Yep, should be 'want'!

Reply:
assuming the same eta for x and y

Reply:
I've removed this example, replacing it with a figure :slightly_smiling_face:

Reply:
we can also keep both..?